### PR TITLE
Feature/arabic support

### PR DESCRIPTION
**Pull request by aabounegm**
Known issues:
- RTL is not properly configured if an RTL language is set on page load (only when using Language Selector component).
- Vuetify strings are not yet translated

### CODE REVIEW

File: components/FooterBar.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-footer absolute app color="#202020">
+  <v-footer absolute app color="#202020" style="display: block;">
```
Comment (enghamzasalem) [inline components/FooterBar.vue:2]: prefer to use class not inline styles

File: nuxt.config.js [hunk 2]
```diff
@@ -50,12 +50,14 @@ export default {
       'nuxt-i18n',
       {
         locales: [
-          { code: 'en', file: 'en.json', name: 'English', shortName: 'Eng' },
-          { code: 'ru', file: 'ru.json', name: 'Русский', shortName: 'Рус' },
+          { code: 'en', file: 'en.json', name: 'English', shortName: 'Eng', rtl: false },
+          { code: 'ru', file: 'ru.json', name: 'Русский', shortName: 'Рус', rtl: false },
+          { code: 'ar', file: 'ar.json', name: 'العربية', shortName: 'عربي', rtl: true },
```
Comment (enghamzasalem) [inline nuxt.config.js:55]: should we add tatar @Yasinaaa ?

File: components/FooterBar.vue (hunk 1 above)
Comment (aabounegm) [reply components/FooterBar.vue:2]: Generally, yes. but here it was an exception and I just wanted to override one property that _sometimes_ caused problems, and it didn't seem to be worth a class 😅

File: nuxt.config.js (hunk 2 above)
Comment (aabounegm) [reply nuxt.config.js:55]: I believe that she once said we will, but I don't know Tatar and this PR is only for Arabic 😛

**Pull request merged**